geomstats/geometry/lie_algebra.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, dimension, n):
         dimension: int
             The dimension of the Lie algebra as a real vector space
         n: int
-            The amount of rows and columns in the matrx representation of the
+            The amount of rows and columns in the matrix representation of the
             Lie algebra
```

Comment:
"." missing at the end of the parameter descriptions.

(Sorry, I know it's not your code, thanks for cleaning it already!)